QATCH/tools/offline_calcs/Injection Force Calculator - QATCH Technologies_files/css2.css

```
@@ -0,0 +1,450 @@
+/* cyrillic-ext */
+@font-face {
+  font-family: 'Montserrat';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Montserrat" (font-family-name-quotes)

(font-family-name-quotes)